hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -94,7 +94,8 @@
 	private String lastUsedExportDir = UNKNOWN_EXPORT_DIR;
 	private BalancesSummary summary;
 
-	Instant periodEnd = NEVER;
+	Instant periodEnd = Instant.now();
+	private int exportPeriod = -1;
```

Comment:
Should use a meaningful default value that could really be used in the calculations below.

Reply:
Yeah, make sense. Will change to default export value.

Reply:
We can even make this `private final int exportPeriod` since it must be set in the constructor.

Reply:
Yeah. Changed.